package.json

```diff
@@ -2,6 +2,7 @@
   "name": "edx",
   "version": "0.1.0",
   "dependencies": {
+    "@edx/brand": "git+https://github.com/edx/brand-edx.org.git",
```

Comment:
Should this default to an open edx version?

Reply:
I don't think so since we have no mechanism for npm overrides in here as far as I know.

Reply:
Not to be difficult - are there legal reasons we don't want edx.org branding (specifically the logo) in here by default?  I thought Open edX instances weren't supposed to start up with edX/Open edX logos by default...

Reply:
By default this package won't be used unless a user has installed and uses the edx.org-next theme